packages/@aws-cdk/aws-ecs-patterns/test/fargate/integ.load-balanced-fargate-service.expected.json

```diff
@@ -545,7 +545,7 @@
       "Properties": {
         "Code": {
           "S3Bucket": {
-            "Ref": "AssetParameters0b9723d54b7db3fbfc1a143398b02392e5fe080a68535480782a949b4372d0f0S3Bucket0997A4A0"
+            "Ref": "AssetParameters01b2187f99280c53b7d58040d494b5d051e1e253601fc32dee62ba56712db247S3Bucket3747EA0C"
```

Comment:
why did this test need to be updated? shouldn't the behavior remain the same if the endpoint was not provided?

Reply:
I think it is because that `ecs-patterns` uses `DnsValidatedCertificate`, please see https://github.com/aws/aws-cdk/blob/7f8c90decd58d2016611e6f94fc506156e047b59/packages/%40aws-cdk/aws-ecs-patterns/lib/base/application-load-balanced-service-base.ts#L338

Test cases failed without this change.

Reply:
And the hash values seem to be generated from source, please see
https://github.com/aws/aws-cdk/blob/b52b43ddfea0398b3f6e05002bf5b97bc831d1a7/packages/%40aws-cdk/core/lib/private/asset-parameters.ts#L16

https://github.com/aws/aws-cdk/blob/b52b43ddfea0398b3f6e05002bf5b97bc831d1a7/packages/%40aws-cdk/core/lib/stack.ts#L540

Reply:
yup, you're right, it's the hash of the Lambda function we're modifying in this PR. LGTM!